classes/Product.php

```diff
@@ -6091,6 +6091,8 @@ public static function getAllCustomizedDatas($id_cart, $id_lang = null, $only_in
     }
 
     /**
+     * @deprecated since 8.0.0, the customization price impact is already included in Product::getPriceStatic
+     *
```

Comment:
Could you deprecate the method using the method explained in devdocs (https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations) ?

Reply:
Resolved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/3d20bbc2854ff0063ff10280afdcb0f1464f5d3b